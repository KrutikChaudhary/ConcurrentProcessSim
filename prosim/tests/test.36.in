2 3
Proc1 6 -1
DOOP 2
LOOP 5
  BLOCK 5
  DOOP 1
END
HALT

Proc2 6 -1
DOOP 2
LOOP 4
  BLOCK 6
  DOOP 2
END
HALT

