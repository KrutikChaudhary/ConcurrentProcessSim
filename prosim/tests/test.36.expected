00000: process 1 new
00000: process 1 ready
00000: process 2 new
00000: process 2 ready
00000: process 1 running
00002: process 1 blocked
00002: process 2 running
00004: process 2 blocked
00007: process 1 ready
00007: process 1 running
00008: process 1 blocked
00010: process 2 ready
00010: process 2 running
00012: process 2 blocked
00013: process 1 ready
00013: process 1 running
00014: process 1 blocked
00018: process 2 ready
00018: process 2 running
00019: process 1 ready
00019: process 2 ready
00019: process 1 running
00020: process 1 blocked
00020: process 2 running
00021: process 2 blocked
00025: process 1 ready
00025: process 1 running
00026: process 1 blocked
00027: process 2 ready
00027: process 2 running
00029: process 2 blocked
00031: process 1 ready
00031: process 1 running
00032: process 1 finished
00035: process 2 ready
00035: process 2 running
00037: process 2 finished
Process 1: Run time 7, Block time 25, Wait time 0
Process 2: Run time 10, Block time 24, Wait time 3
